### PR TITLE
Update to 64.2

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/icu @conda-forge/arm-arch  This is only failing due to https://github.com/conda/conda-build/issues/3583. Should be pin `conda-build` manually to `<3.18` in the feedstock sources or do we want to merge without Aarch64 for now?

**Comment by mariusvniekerk** [comment]
@xhochy lets just do that for now.  rerender will fix it eventually

**Comment by xhochy** [comment]
@conda-forge/icu This is ready for review/merge.

File: .azure-pipelines/build_steps.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ conda-build:
 
 CONDARC
 
-conda install --yes --quiet conda-forge-ci-setup=2 conda-build -c conda-forge
+conda install --yes --quiet conda-forge-ci-setup=2 conda-build=3.17.8 -c conda-forge
```
Comment (ocefpaf) [inline .azure-pipelines/build_steps.sh]: Argh!

**Comment by ocefpaf** [comment]
Thanks @xhochy!

**Pull request merged**